actors/evm/tests/contracts/FilecoinFallback.sol

```diff
@@ -2,16 +2,16 @@
 pragma solidity >=0.4.25 <=0.8.17;
 
 contract FilecoinFallback {
-    function handle_filecoin_method(uint64 method, uint64 codec, bytes calldata params) pure public returns (uint32, uint64, bytes memory) {
```

Comment:
are you sure?

Reply:
It's not necessary -- it wasn't compiling with an older solc, because there used to be an assertion that `calldata` is only used for `external` functions (not `public`). This was dropped in solidity 0.8, but I think it's still correcter to use `memory` here.

But it's not necessary so I'll drop.

Reply:
Ah, well, this function should probably be marked as "external" in that case.